examples/sampling/graphbolt/link_prediction.py

```diff
@@ -303,11 +359,6 @@ def train(args, graph, features, train_set, valid_set, model):
             if step + 1 == args.early_stop:
```

Comment:
So how do we benchmark the time performence of training

Reply:
Sorry, I forget to add the corresponding the code of time cost calculation.